### PR TITLE
Switching deprecated commands with non deprecated

### PR DESCRIPTION
**Pull request by YuvHayun**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Description
Switched the usage of multiple deprecated integration commands with non-deprecated ones.

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

### CODE REVIEW

File: Packs/RsaNetwitnessSecurityAnalytics/ReleaseNotes/1_0_6.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+
+#### Scripts
+##### NetwitnessSAListIncidents
+- Fixed an issue where ***nw-list-incidents*** which is deprecated was used.
```
Comment (ShahafBenYakir) [inline Packs/RsaNetwitnessSecurityAnalytics/ReleaseNotes/1_0_6.md:4]: Let's add the replacement.
```suggestion
- Fixed an issue where ***nw-list-incidents*** which is deprecated was used, was replaced by ***netwitness-im-list-incidents*** .
```

File: Packs/RsaNetwitnessSecurityAnalytics/ReleaseNotes/1_0_6.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+- Fixed an issue where ***nw-get-events*** which is deprecated was used.
+- Updated the Docker image to: *demisto/python:2.7.18.27799*.
+##### NetwitnessSAAddEventsToIncident
+- Fixed an issue where ***nw-add-events-to-incident*** which is deprecated was used.
+- Updated the Docker image to: *demisto/python:2.7.18.27799*.
+##### NetwitnessSAGetComponents
+- Fixed an issue where ***nw-get-components*** which is deprecated was used.
+- Updated the Docker image to: *demisto/python:2.7.18.27799*.
+##### NetwitnessSACreateIncident
+- Fixed an issue where ***nw-create-incident*** which is deprecated was used.0).
```
Comment (ShahafBenYakir) [inline Packs/RsaNetwitnessSecurityAnalytics/ReleaseNotes/1_0_6.md:19]:
```suggestion
- Fixed an issue where ***nw-create-incident*** which is deprecated was used.
```

**Review by ShahafBenYakir — APPROVED** [review]
Fix my rn comment and merge

**Comment by YuvHayun** [comment]
@ShahafBenYakir will need force merge here due to [CJ105] error.

**Pull request merged**